src/transactions/TransactionFrame.cpp

```diff
@@ -558,8 +558,9 @@ TransactionFrame::applyOperations(SignatureChecker& signatureChecker,
             }
             if (!errorEncountered)
             {
+                auto header = ls.loadHeader();
```

Comment:
we're probably missing some checks somewhere as this is wrong:
it should be `lsOperation.loadHeader()`